hack/csi_client/pkg/csi/client.go

```diff
@@ -14,21 +14,15 @@ import (
 )
 
 const (
-	attrIP           = "ip"
-	attrVolume       = "volume"
-	smbUser          = "smbUser"
-	smbPassword      = "smbPassword"
 	reqNodePublish   = "nodepublish"
 	reqNodeUnpublish = "nodeunpublish"
 )
 
 var (
 	goOs          = runtime.GOOS
-	errHost       = fmt.Errorf("host name is required")
-	errVolume     = fmt.Errorf("volume name is required")
+	errVolumeAttr = fmt.Errorf("volume attributes are required")
```

Comment:
volumeAttr and secret errs are not used either.  Should we error for empty targetPath?

Reply:
True, I'll just remove all errors. Since we're leaving all validation to the driver, I don't think erroring on an empty targetPath is necessary. The driver will return an error on empty targetPath.